packages/discovery-react-components/src/components/CIDocument/components/CIDocument/CIDocument.tsx

```diff
@@ -611,13 +632,21 @@ function getActiveIndex(activeIds: string[], highlightedList: any[]): number {
 
 function getActiveElement(activeIds: string[], itemList: any[]): any {
   return itemList.find(item => {
-    if (item.allAttributeIds) {
+    if ('allAttributeIds' in item) {
       return isEqual(item.allAttributeIds.sort(), activeIds.sort());
+    } else if (isEqualId(item, activeIds)) {
+      return true;
+    } else if ('attributes' in item) {
+      return item.attributes.find((attribute: any) => isEqualId(attribute, activeIds));
```

Comment:
since all of these `else-ifs` have return statements, I think it would be more readable to just separate them in their own `if statements instead` of chaining them like above, or maybe use a switch statement. This is just a personal preference of mine, if you want to keep it this way that's fine too

Reply:
Yah I prefer if- elseIf chaining pattern.